content/blog/overview-of-the-platform-level-data-model-for-redfish®-device-enablement-standard.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Overview of the Platform Level Data Model for Redfish® Device Enablement
+  standard
+date: 2022-05-12T16:04:30.750Z
+author: François Donzé
+authorimage: /img/fdz-photoprofile.png
+tags:
```

Comment:
Please add the tag ilo-restful-api to have the blog post tied with the platform page "ILO Redfish API".

Reply:
Done